src/logical/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (number1,number2, operator) => {
+    if (number2 === undefined) return number1;
```

Comment:
**[OPTIONAL]**
- I wonder the purpose of this line, but let see how it would turn out🤔 .